reference/markdown/Configuration-options.md

```diff
@@ -0,0 +1,22 @@
+You can create a custom plug-in to set different configuration options for Markdown parsing and conversion to DITA.
```

Comment:
Maybe this topic should be merged into the _Custom schema configuration_ that needs to be extracted from _Markdown schema_ topic. You can only use these from Java code.

Reply:
Resolved in ae667eb.